scripts/redcap/wrong_date_associations.py

```diff
@@ -148,7 +150,41 @@ def log_row(row: pd.Series, **kwargs):
     for _, row in errors_df.reset_index().iterrows():
         log_row(row, uid=uid_template, **kwargs)
 
+def get_special_cases(marks, session):
```

Comment:
Given the context of its use, `subtract_special_cases_from_marks` would be a more descriptive name.

Reply:
Changed in the next commit.